### PR TITLE
mimic mailmaps

### PR DESCRIPTION
**Pull request by theanalyst**
mailmap updates through luminous and mimic

### CODE REVIEW

**Comment by theanalyst** [comment]
Mailmaps I had locally since Luminous, I'll create another one with larger mimic changeset

**Comment by theanalyst** [comment]
sample output with the current mailmap https://gist.github.com/theanalyst/ab71736b768a7073c4008613eda30203

File: .mailmap [hunk 1]
```diff
@@ -441,12 +465,15 @@ Wei Qian <weiq@dtdream.com>
 Wenjun Huang <wenjunhuang@tencent.com>
 Wido den Hollander <wido@42on.com> <wido@widodh.nl>
 Wu Xingyi <wuxingyi@letv.com>
+Wuming Qiao <wumingqiao@inspur.com>
```
[the remark this answers is not in the document]
Comment (wumingqiao) [reply .mailmap]: Yes, my name is Mingqiao Wu

[the remark this answers is not in the document]
Comment (theanalyst) [reply .mailmap]: ack thanks correcting

**Comment by batrick** [comment]
@theanalyst pleaes update

**Comment by liewegas** [comment]
needs rebase?

**Comment by liewegas** [comment]
needs rebase

**Pull request closed without merging**